src/routes/age_groups.py

```diff
@@ -0,0 +1,112 @@
+from fastapi import APIRouter
+from typing import List
+from starlette.status import HTTP_200_OK, \
+    HTTP_201_CREATED, HTTP_400_BAD_REQUEST, \
+    HTTP_404_NOT_FOUND
```

Comment:
Same, try to use parenthesis
```suggestion
from starlette.status import (
    HTTP_200_OK,
    HTTP_201_CREATED,
    HTTP_400_BAD_REQUEST,
    HTTP_404_NOT_FOUND
)
```